samza-core/src/main/scala/org/apache/samza/coordinator/JobModelManager.scala

```diff
@@ -104,7 +107,15 @@ object JobModelManager extends Logging {
 
     val streamMetadataCache = new StreamMetadataCache(systemAdmins = systemAdmins, cacheTTLms = 0)
     val previousChangelogPartitionMapping = changelogManager.readChangeLogPartitionMapping()
-    val jobModelManager = getJobModelManager(config, previousChangelogPartitionMapping, localityManager, streamMetadataCache, null)
+
+    val processorList = new ListBuffer[String]()
+    val containerCount = new JobConfig(config).getContainerCount
+    for (i <- 0 to containerCount) {
```

Comment:
I think you need to use `until` here. 
`to` is inclusive of both the ranges `[from, to]`

Reply:
Yup. Realized this while testing and fixed it.